l10n_co_trm/models/res_currency.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import time
+
+import requests
+from lxml import etree
+
+from odoo import api, models
+
+_logger = logging.getLogger(__name__)
+
+
+class ResCurrency(models.Model):
+    _inherit = "res.currency"
+
+    def get_rate(self):
+        try:
+            date_now = time.strftime("%Y-%m-%d")
+            wsdl = "https://www.superfinanciera.gov.co/SuperfinancieraWebServiceTRM/TCRMServicesWebService/TCRMServicesWebService?WSDL"
+            headers = {"content-type": "text/xml"}
+            body = f"""
+            <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:act="http://action.trm.services.generic.action.superfinanciera.nexura.sc.com.co/">
+               <soapenv:Header/>
+               <soapenv:Body>
+                  <act:queryTCRM>
+                     <!--Optional:-->
+                     <tcrmQueryAssociatedDate>{date_now}</tcrmQueryAssociatedDate>
+                  </act:queryTCRM>
+               </soapenv:Body>
+            </soapenv:Envelope>
+            """
+            response = requests.post(wsdl, data=body, headers=headers, timeout=10)
+            rate = float(etree.XML(response.content).xpath("//value")[0].text)
+        except Exception as e:
+            _logger.exception(e)
+            return str(e)
+        _logger.info(f"TRM vigente en la fecha {date_now} es de: {rate} COP")
+        return rate
+
+    @api.model
+    def get_trm(self):
+        rate = self.get_rate()
+        # rate_amount = 1 / rate
+        date_now = time.strftime("%Y-%m-%d")
```

Comment:
Suggestion: rename to `date_today`? `now`makes me think is is a datetime.
```suggestion
        date_today = time.strftime("%Y-%m-%d")
```

Reply:
done.